sdk/examples/intkey_python/sawtooth_intkey/client_cli/intkey_workload.py

```diff
@@ -89,8 +92,22 @@ def __init__(self, delegate, args):
         self._delegate = delegate
         self._deps = {}
         context = create_context('secp256k1')
-        self._signer = CryptoFactory(context).new_signer(
-            context.new_random_private_key())
+        crypto_factory = CryptoFactory(context=context)
+        if args.key_file is not None:
+            try:
+                with open(args.key_file, 'r') as infile:
+                    signing_key = infile.read().strip()
+                private_key = Secp256k1PrivateKey.from_hex(signing_key)
+
+                self._signer = crypto_factory.new_signer(
+                    private_key=private_key)
+            except ParseError:
+                self._signer = crypto_factory.new_signer(
```

Comment:
Same situation as previously mentioned: the user provided a bad key and they should be informed of that error.  Otherwise, it will silently go on to create a new key.

Reply:
Makes sense. I will log an error and not make the key.

Reply:
Since we don't have an IntkeyCliError, I think it makes sense to just not catch the error.